src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php

```diff
@@ -68,6 +68,10 @@ public function process(ContainerBuilder $container)
                         '/[^a-z0-9]/i',
                     ), function ($matches) { return strtoupper($matches[0]); }, $event['event']);
```

Comment:
I didn't know there is such magic default logic.

Reply:
true, yet convenient enough to keep IMHO, or at least didnt want to spoil this feat. along with a deprecation.